.clj-kondo/config.edn

```diff
@@ -0,0 +1 @@
+{:config-paths ["../clj-kondo/clj-kondo.exports/clara/rules"]}
```

Comment:
Why not put the config inline here instead of the indirection?

Reply:
This is inline with export/import functionality, the main point of this PR is not just to lint clara-rules (the project) itself but to add support for linting out of the box for the library. This is how the exported lint hooks are integrated with the project, I followed the export/import guide: https://github.com/clj-kondo/clj-kondo/blob/master/doc/config.md#exporting